etc/lmod-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if [ $# = 0 ]; then
+   L_MACHINE=${MACHINE}
+   cat << EOF_USAGE
+Usage: source etc/lmod-setup.sh PLATFORM
+
+OPTIONS:
+   PLATFORM - name of machine you are building on
+      (e.g. cheyenne | hera | jet | orion | wcoss2 )
```

Comment:
I would also remove this reference to Cheyenne.

Reply:
Done